lib/icinga/externalcommandprocessor.cpp

```diff
@@ -24,6 +24,7 @@
 
 using namespace icinga;
 
+bool ExternalCommandProcessor::m_DownTimeRemoved;
```

Comment:
Why did you put this here instead of a local variable? All uses of it look like a local variable should suffice. Or is there any sharing between functions that I'm missing?